docs/adding_a_new_microservice.md

```diff
@@ -1,95 +1,136 @@
 ## Adding a new Microservice to Metaphysics
 
+Ideal place we want to be: You don't need to do this because you're using GraphQL and Stitching your schema. However, if
+you need to add in a service manually.
```

Comment:
I think this is an incomplete sentence, maybe: 

...if you need to add in a service manually follow this guide.